content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -54,99 +54,86 @@ In this example:
-  {{< /note >}}
+    {{< note >}}
+    The `matchLabels` field is a map of {key,value} pairs. A single {key,value} in the `matchLabels` map
+    is equivalent to an element of `matchExpressions`, whose key field is "key" the operator is "In",
+    and the values array contains only "value".
+    All of the requirements, from both `matchLabels` and `matchExpressions` are ANDed together – they must all be
+    satisfied in order to match.
+    {{< /note >}}
 
 * The `template` field contains the following sub-fields:
   * The Pods are labeled `app: nginx`using the `labels` field.
   * The Pod template's specification, or `.template.spec` field, indicates that
   the Pods run one container, `nginx`, which runs the `nginx`
   [Docker Hub](https://hub.docker.com/) image at version 1.7.9.
   * Create one container and name it `nginx` using the `name` field.
-  * Open port `80` so that the container can send and accept traffic.
-
-To create this Deployment, run the following command:
-
-```shell
-kubectl apply -f https://k8s.io/examples/controllers/nginx-deployment.yaml
-```
-
-{{< note >}}
-You may specify the `--record` flag to write the command executed in the resource annotation `kubernetes.io/change-cause`. It is useful for future introspection, for example to see the commands executed in each Deployment revision.
-{{< /note >}}
-
-Next, run `kubectl get deployments`. The output is similar to the following:
-
-```shell
-NAME               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-nginx-deployment   3         0         0            0           1s
-```
-
-When you inspect the Deployments in your cluster, the following fields are displayed:
 
-* `NAME` lists the names of the Deployments in the cluster.
-* `DESIRED` displays the desired number of _replicas_ of the application, which
-   you define when you create the Deployment. This is the _desired state_.
-* `CURRENT` displays how many replicas are currently running.
-* `UP-TO-DATE` displays the number of replicas that have been updated to achieve
-   the desired state.
-* `AVAILABLE` displays how many replicas of the application are available to
-   your users.
-* `AGE` displays the amount of time that the application has been running.
+  Follow the steps given below to create the above Deployment:
+
+  Before you begin, make sure your Kubernetes cluster is up and running.
+
+  1. Create the Deployment, run the following command:
+
+      {{< note >}}
+      You may specify the `--record` flag to write the command executed in the resource annotation `kubernetes.io/change-cause`. It is useful for future introspection.
+      For example, to see the commands executed in each Deployment revision.
+      {{< /note >}}
+
+    ```shell
+    kubectl apply -f https://k8s.io/examples/controllers/nginx-deployment.yaml
+    ```
+
+  2. Next, let's run `kubectl get deployments` to check if the Deployment is created. If the Deployment is still being created, the output is similar to the following :
```

Comment:
Per the style guide, [avoid using we](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we), plus grammar nits: 
> 2. Run `kubectl get deployments` to check if the Deployment was created. If the Deployment is still being created, the output is similar to the following: